src/behaviors/Behavior.js

```diff
@@ -19,10 +19,6 @@ var noExportProperties = [
     'treeColumnPropertiesColumnSelection',
 ];
 
-var isNull = {
-    isNull: true
-};
```

Comment:
I eliminated this isNull nonsense. Just returns undefined now. This is really only consumed by renderer. However, we will list it in our "breaking changes" doc for 1.1.0.